kernel/common/ezautolinkoperator.php

```diff
@@ -10,6 +10,13 @@
 
 class eZAutoLinkOperator
 {
+    static $max = 72;
+    
+    static $isLinkWwwUrl = false;
+    
+    static $hideProtoclText = '';
```

Comment:
This variable name should also be changed to `$hideProtocolsText` at the very least. We think this was merely a spelling typo. We request you change this variable name.